components/_components.scss

```diff
@@ -33,3 +33,130 @@
     border-radius: 50%;
   }
 }
+
+.mail {
+  display: inline-block;
+  transform: translateY(-2px);
+  color: $color-grey;
+}
+
+.center-card {
+  display: grid;
+  grid-template-columns: auto 1fr;
+
+  .profile-image.card-image {
+    grid-row: 1/5;
+    grid-column: 1/2;
+  }
+
+  &-title {
+    display: grid;
+    grid-template-columns: auto auto 1fr auto;
+    margin-top: 1rem;
+    &-icon {
```

Comment:
I would like to know how you do your sass . The way I do it is very cumbersome, I end up with very long class names